tests/app/main/test_suppliers.py

```diff
@@ -409,11 +409,13 @@ def test_should_be_an_error_if_duns_number_in_use(self, data_api_client):
         res = self.client.post(
             "/suppliers/duns-number",
             data={
-                'duns_number': "9999999999"
+                'duns_number': "123456789"
             }
         )
         assert_equal(res.status_code, 400)
-        assert_true("DUNS Number must be 9 digits" in res.get_data(as_text=True))
+        page = res.get_data(as_text=True)
+        assert_true("A supplier account already exists with that DUNS number" in page)
```

Comment:
If the tests are passing then you don't see any of the mess, and if they fail then maybe you do need to sift through the output to find the error?  Otherwise with failing tests you have to stick a `print` in there to work out what's gone wrong.

So I vote to go with `assert_in`, even though I see your point.